### ISSUE TITLE
Does not allow you to use Siri for things like sending messages

### ISSUE THREAD

**Issue by NSExceptional** [Muirey03/SmallSiri#3]
You can't see whatever Siri shows on screen such as a message confirmation box. It makes Siri all but useless. There needs to be a way to expand the window.

**Comment by magicman0978** [comment]
I completely agree,  ideally it should be automatically adaptable  but after would be virtually impossible for anyone to pull that off save Apple  I would settle for the ability to tap and then tap out of an expanded view. Something like an  slightly taller expanded AirPods overlay within the Siri page running in it.
